pages/index.js

```diff
@@ -1,3 +1,112 @@
+import styled from "styled-components";
+import Image from "next/image";
+
+import products from "@/products.json";
+
 export default function HomePage() {
-  return <h1>Hello from Next.js</h1>;
+  return (
+    <AppContainer>
+      <ShopName>Shop Name</ShopName>
+      <GallerySection>
+        {/* TODO: refactor component in the next user story */}
+        <GalleryList>
+          {products.map((product) => {
+            return (
+              <Card key={product.id}>
+                <StyedThumbnail
+                  src={product.thumbnail}
+                  alt={product.title}
+                  width={120}
+                  height={152}
+                />
+                <ItemInformation>
+                  <ItemName>{product.title}</ItemName>
+                  <ItemPrice>{product.price}€</ItemPrice>
+                </ItemInformation>
+              </Card>
+            );
+          })}
+        </GalleryList>
+      </GallerySection>
+    </AppContainer>
+  );
 }
+
+// Not the final styling.
+// That's just the styling for User Story 1
+// Will add more styling to future User Story to simply it.
+// Real styling comes much later
+const AppContainer = styled.div`
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: center;
+  border: solid blue;
+  height: auto;
+  width: 375px;
```

Comment:
You should not set the width of your app container to a fixed size, it should stay responsive. You can preview the app in the mobile view in your browser.